### DISCUSSION TITLE
Starter workflow is not suggested for repos within the organization

### DISCUSSION THREAD

**Discussion by white-eagle-83** [community/community#55868] [Enterprise]
### Select Topic Area

Question

### Body

I have successfully added the starter workflow as specified in the guide [here](https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization). But I am unable to use it.  
![Screenshot 2023-05-18 at 14 15 36](https://github.com/community/community/assets/98066757/bdbfd550-c2f4-42b9-8f86-7dcc6e0449a7)
![Screenshot 2023-05-18 at 14 15 44](https://github.com/community/community/assets/98066757/dc13ccd0-2362-4b33-a63e-4cb9fc383746)

I followed the [troubleshooting link](https://docs.github.com/en/enterprise-server@3.7/admin/github-actions/advanced-configuration-and-troubleshooting/troubleshooting-github-actions-for-your-enterprise#bundled-actions) from the page but it did not help. 

One additional thing to note, our internal gihub(enterprise) has `actions` org. It did not have `starter-workflows` repo within it. Since it was present on our internal testing github and this testing github is able to suggest the starter workflows; thinkng that this missing configuration could be the reason the suggestions are not populated for the new workflows, I copied the [public repo data](https://github.com/actions/starter-workflows) to the internal repo but even that did not help. 

What I could be missing getting the starter worklow suggestions turned on? Is there any additional config setting that needs to be enabled?

**Comment by brignano** [comment]
> I copied the public repo data to the internal repo but even that did not help.

I think this should have resolved your issue however, make sure you copied the [`ghes`](https://github.com/actions/starter-workflows/tree/ghes) branch.

I observed this behavior when I deleted `actions/starter-workflows` in GHES 3.12.

Restoring `actions/starter-workflows` resolved the issue and my reusable workflows were displayed as expected.
